### DISCUSSION TITLE
VIC20 No audio over HDMI

### DISCUSSION THREAD

**Discussion by TheRetroChannel** [frntc/Sidekick64#63] [Q&A] [answered]
Hi all, I built the VIC20 adaptor and hooked up the A15 address line, but I'm not getting any audio over HDMI. The HDMI video looks perfect, just no audio. I've already tried the same with another VIC20 and got the same result. 

I'm using a Pi 3A+ and audio does work correctly when running something like retropie on it so I don't think it's the Pi itself. I've also tried the different overclock settings on the Pi but the result is always the same. Any ideas here?

Cheers,

Mark

**Comment by frntc** [comment, marked as answer]
Do you have the possibility to try another HDMI device? I didn't experience no sound, but not every HDMI device seems to handle the audio stream (of/with this particular HDMI output in Circle) equally well.

**Reply by TheRetroChannel** [reply]
Yes you are correct. The 3 HDMI devices I tried it with just happened to be the 3 that didn't accept the audio. Trying it with other displays works perfect. Thank you Frenetic

**Reply by frntc** [reply]
thanks for testing + confirmation!
